scmgr/commands.go

```diff
@@ -0,0 +1,168 @@
+package main
+
+import cli "gopkg.in/urfave/cli.v1"
+
+func getCommands() []cli.Command {
+	groupsDef := "[group-definition]"
+	return []cli.Command{
+		{
+			Name:  "admin",
+			Usage: "administer the skipchain-service",
+			Subcommands: []cli.Command{
+				{
+					Name:    "link",
```

Comment:
I think there needs to be a "unlink" command, because knowing that you can delete the scmgr config file is pretty arcane knowledge.